source/Ship.cpp

```diff
@@ -687,6 +716,22 @@ void Ship::Save(DataWriter &out) const
 			out.Write("destination system", targetSystem->Name());
 		if(isParked)
 			out.Write("parked");
+		if(!cargo.IsEmpty())
+		{
+			out.Write("cargo hold");
+			out.BeginChild();
+			{
+				for(const auto &it : cargo.MissionCargo())
+					out.Write("mission", it.first->Name(), "cargo", it.second);
+				for(const auto &it : cargo.PassengerList())
+					out.Write("mission", it.first->Name(), "passengers", it.second);
+				for(const auto &it : cargo.Outfits())
```

Comment:
Aren't we already writing outfits and commodities with `cargo.Save(out)`?

Reply:
this is corrected